### PR TITLE
fix: tweak heading > code font-size

### PR DESCRIPTION
**Pull request by zchsh**
## 🔗 Relevant links

- [Preview link][preview] 🔎
- [Asana task][task] 🎟️

## 🗒️ What

Tweaks font size of inline code within content headings.

## 🛠️ How

Adjusts font-size for inline `<code />` in content headings from `0.6em` to `0.8em`.

### Before

inline-code-before

### After

inline-code-after

## 🧪 Testing

- [ ] Visit a docs page with inline code in a heading, such as [/vault/docs/configuration/listener/tcp][preview], and ensure it looks as expected

[task]: https://app.asana.com/0/1202114367927919/1202439831418480/f
[preview]: https://dev-portal-git-zstweak-inline-code-hashicorp.vercel.app/vault/docs/configuration/listener/tcp

### CODE REVIEW

File: src/components/dev-dot-content/dev-dot-content.module.css [hunk 1]
```diff
@@ -72,7 +72,7 @@
     background-color: var(--token-color-palette-neutral-50);
     border-radius: 5px;
     border: 1px solid rgba(101, 106, 118, 0.2);
-    font-size: 13px;
+    font-size: 0.8125rem;
```
Comment (zchsh) [inline src/components/dev-dot-content/dev-dot-content.module.css:75]: Bit of a random tangent, but figured might as well use an equivalent `rem` value here to match practices throughout the rest of the project.

File: src/components/dev-dot-content/dev-dot-content.module.css [hunk 2]
```diff
@@ -217,7 +217,7 @@
   less jarring.
   */
   & code {
-    font-size: 0.6em;
+    font-size: 0.8em;
```
Comment (zchsh) [inline src/components/dev-dot-content/dev-dot-content.module.css:220]: Bumps to `0.8em`, as specific in [the task](https://app.asana.com/0/1202114367927919/1202439831418480/f)

**Review by BRKalow — APPROVED** [review]
👍  🔍

**Review by braydenlove — APPROVED** [review]
SHIP IT

**Pull request merged**